server/channels/store/sqlstore/file_info_store.go

```diff
@@ -713,9 +713,10 @@ func (fs SqlFileInfoStore) CountAll() (int64, error) {
 	return count, nil
 }
 
-func (fs SqlFileInfoStore) GetFilesBatchForIndexing(startTime int64, startFileID string, limit int) ([]*model.FileForIndexing, error) {
+func (fs SqlFileInfoStore) GetFilesBatchForIndexing(startTime int64, startFileID string, includeDeleted bool, limit int) ([]*model.FileForIndexing, error) {
```

Comment:
0/5, but should we consider reverting this change and just getting these back by default as before? (Might make it easier to revert this PR vs. hand removing the job later.)

Reply:
I am actually considering keeping this flag in case it becomes useful later, unless you have strong opinions.

Reply:
No objection to keeping the flag, and no strong objection to keeping it in this PR, just thinking ahead to the revert when this job finishes. Happy to defer to you :)